go/libkb/id_table.go

```diff
@@ -1359,7 +1366,14 @@ func (idt *IdentityTable) proofRemoteCheck(ctx context.Context, hasPreviousTrack
 		return
 	}
 
-	res.err = pc.CheckStatus(idt.G().CloneWithNetContext(ctx), *res.hint)
+	// ProofCheckerModeActive or Passive mainly decides whether we need to reach out to
+	// self-hosted services. We want to avoid so doing when the user is acting passively
+	// (such as when receiving a message).
+	pcm := ProofCheckerModePassive
+	if (hasPreviousTrack && res.trackedProofState != keybase1.ProofState_NONE) || itm == IdentifyTableModeActive {
```

Comment:
Should probably also make sure it's not `ProofStatus_UNCHECKED` before switching to active.